app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/Makefile.old b/Makefile
+index a4e4285..38e2ff8 100644
+--- a/Makefile.old
++++ b/Makefile
+@@ -39,9 +39,9 @@ stamp-unpack-git:
+ 	touch $@
+ 
+ setup: stamp-setup
+-stamp-setup: stamp-unpack
+-	$(MAKE) -C perkamon setup
+-	mv perkamon/build .
++stamp-setup:
++	$(MAKE) -C man-pages-$(PERKVERSION) setup
```

Comment:
But why do you need to change it? Wouldn't it be simpler to just rename the directory after unpacking?